src/igvfd/schemas/in_vitro.json

```diff
@@ -0,0 +1,106 @@
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/alternate_accessions"
+        },
+        {
+            "$ref": "mixins.json#/accessioned_status"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/submitter_comment"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "organoid.json#/dependentSchemas"
+        ]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "post_differentiation_time": {
+            "title": "Post-differentiation time",
+            "description": "The time that elapsed past the point when biosamples were differentiated.",
+            "comment": "Use in conjunction with post_differentiation_time_units to specify time elapsed post-differentiation. Use of this property requires post_differentiation_time_units specification.",
+            "type": "integer"
+        },
+        "post_differentiation_time_units": {
+            "title": "Post-differentiation time units",
+            "description": "The units of time that elapsed past the point when biosamples were differentiated.",
+            "comment": "Use in conjunction with post_differentiation_time to specify time elapsed post-differentiation. Use of this property requires post_differentiation_time specification.",
+            "type": "string",
+            "enum": [
+                "minute",
+                "hour",
+                "day",
+                "week",
+                "month",
+                "stage"
+            ]
+        },
+        "date_obtained": {
+            "title": "Date collected"
+        },
+        "treatments": {
+            "title": "Differentiation treatment",
+            "description": "A list of treatments applied to the biosample.",
+            "comment": "A list of treatments applied to the biosample.",
+            "type": "array"
```

Comment:
I am surprised this works.  I thought that you had to describe the items within the array.

Reply:
I guess it is alright for now since it is just a placeholder until treatment is merged. unless you have an example of what you'd prefer to set items of the array to?

Reply:
Lets leave this for now